addons/core/translations/resources/en-us.yaml

```diff
@@ -275,9 +275,11 @@ host-set:
     filter:
       label: Filter
       aws:
-        help: Create a filter to select resources using values such as tag name or tag value. The AWS filter format is tag:<tag-name>=<tag-value>. For example:tag:application=production
+        help: 'Create a filter to select resources using values such as tag name or tag value. The AWS filter format is tag:<tag-name>=<tag-value>. For example: tag:application=production'
       azure:
         help: Create a filter to select resources using values such as tag name or tag value. The Azure filter format is:tagName eq ‘<tag-name>’ and tagValue eq ‘<tag-value>'. For example:tagName eq 'application' and tagValue eq 'production'
+      gcp:
+        help: 'Create a filter to select resources using values such as label name or label value. The GCP filter format is labels.<label-name>=<label-value>. For example: labels.application=production'
```

Comment:
Should it say `label.<label-name>=<label-value>` instead of `labels.<label-name>=<label-value`?

Reply:
In this case it actually is `labels`, it would look like `labels.env=dev`

<img width="484" alt="image" src="https://github.com/user-attachments/assets/d7f1a510-8338-4caa-a283-a302f907ae54" />